.github/workflows/tests.yml

```diff
@@ -47,8 +47,27 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Download reference data
-        run: bash .ci-helpers/download_reference_data.sh
+      - name: Clone tardis-sn/tardis-refdata
+        uses: actions/checkout@v2
+        with:
+          repository: tardis-sn/tardis-refdata
+          path: tardis-refdata
+          lfs: false
+
+      - name: Create LFS file list
+        run: git lfs ls-files -l | cut -d' ' -f1 | sort > .lfs-assets-id
+        working-directory: tardis-refdata
+
+      - name: Restore LFS cache
+        uses: actions/cache@v2
+        id: lfs-cache
+        with:
+          path: tardis-refdata/.git/lfs
+          key: ${{ runner.os }}-lfs-${{ hashFiles('tardis-refdata/.lfs-assets-id') }}-v1
+
+      - name: Git LFS Pull
+        run: git lfs pull
+        working-directory: tardis-refdata
```

Comment:
I agree with Ezequiel that it would be more instructive to make this step conditional on the `cache-hit` even if it doesn't make a difference in practice.